server/methods/accounts/accounts.js

```diff
@@ -592,7 +594,7 @@ export function inviteShopOwner(options) {
     });
   }
 
-  const { shopId } = Meteor.call("shop/createShop", userId) || {};
+  const { shopId } = Meteor.call("shop/createShop", userId, shopData) || {};
```

Comment:
`shop/createShop` already accepts shop data, so the major change here is accepting a new parameter and passing it on to the createShop method